Doppler.HtmlEditorApi/Repositories.DopplerDb/DopplerCampaignContentRepository.cs

```diff
@@ -71,11 +72,19 @@ public async Task<CampaignState> GetCampaignState(string accountName, int campai
             : campaignStateData.Status == DopplerCampaignStatusInWinnerInABSelectionProcess ? CampaignStatus.InWinnerInABSelectionProcess
             : CampaignStatus.Other;
 
-        return new CampaignState(
-                campaignStateData.OwnCampaignExists,
+        if (campaignStateData.TestType == 0)
+        {
+            return new ClassicCampaignState(campaignStateData.ContentExists, campaignStateData.EditorType, campaignStatus);
+        }
+
+        var campaignTestABCondition = campaignStateData.TestType is DopplerCampaignTestTypeSubject ? TestABCondition.TypeTestABSubject
+        : TestABCondition.TypeTestABContent;
```

Comment:
The second line is subordinate to the first one, what do you think about adding an indentation?

```suggestion
        var campaignTestABCondition = campaignStateData.TestType is DopplerCampaignTestTypeSubject ? TestABCondition.TypeTestABSubject
            : TestABCondition.TypeTestABContent;
```

Or better yet:


```suggestion
        var campaignTestABCondition = campaignStateData.TestType is DopplerCampaignTestTypeSubject 
            ? TestABCondition.TypeTestABSubject
            : TestABCondition.TypeTestABContent;
```

Reply:
Fixed